### ISSUE TITLE
Crates Or Playlists? 

### ISSUE THREAD

**Issue by raynessupreme** [pideck/pideck#116]
Hello again. 

I recently doubled my USB drive capacity from 32GB to 64GB, so I could have more audio files to hand. I ran the BPM and Key scripts so that they can show, although it only seems that the BPM tags show and not the song Key.

One thing I couldn't find any info on (hence asking here), is whether or not it was possible to organise your audio files by crates or Playlists within xWax? Or possibly have more iD3 tag sections showing (that can also be searchable), such as genre and album title. It also seemed that as much as I updated a whole load of iD3 tags in the process of also creating some 
.m3u Playlists to trial, that xWax wasn't reading the updated file tags and was just displaying the titles of the audio files. 

If anyone has any ideas on supporting this I would be appreciative.

Many Thanks. 

Shaun.

**Comment by danielhjames** [comment]
Hi @raynessupreme in PiDeck we support a simplified import script in which one USB stick equals one crate. There are some modifications to the xwax GUI to hide other crates. This is partly to keep navigation and on-screen-display simple.

However xwax supports all kinds of import scripts including custom scripts, and you can arrange your crates any way you want. Please see for example https://wiki.xwax.org/oligau_s_scan_script

**Comment by GuusR** [comment]
@raynessupreme Have you tried the folders to crates function? It works, but when rescanning the library xwax will scan the old list of folders, so rescanning a different usb stick only works with a full restart of xwax.

@danielhjames Is it possible to have the crates shown in a column like song, bpm etc. are shown? I couldn't find a way to edit which columns are shown/hidden in the GUI.

**Comment by danielhjames** [comment]
@GuusR yes, you can display crates in the left-side column of standard xwax, please see the screenshot here: http://xwax.org/screenshots/xwax-1.3.png
For PiDeck we removed the code for the left column of the GUI so that the GUI fits on small screens. Please see line 1390 onwards in this commit: https://github.com/xwax/xwax/commit/ca456e6eb1c176bd9e64682d75d71b7c1f6130fa
